ui/src/shared/components/ResizeContainer.js

```diff
@@ -97,7 +97,7 @@ class ResizeContainer extends Component {
 
   render() {
     const {bottomHeightPixels, topHeight, bottomHeight, isDragging} = this.state
-    const {containerClass, children} = this.props
+    const {containerClass, children, isKapacitorTheme} = this.props
```

Comment:
Instead of a boolean here, lets have a `theme` prop that is a string called `'kapacitor'`.  This way, should we want to use other themes we wont have to keep track of `n` different bool props, just one theme prop.